google-cloud-bigtable/lib/google/cloud/bigtable/table/column_family_map.rb

```diff
@@ -23,12 +23,12 @@ class Table
         # It is used to create an instance.
         # @example Add column family with name and garbage collection rule
         #
-        #  column_families = Google::Cloud::Bigtable::Instance::ColumnFamilyMap.new
+        #  column_families = Google::Cloud::Bigtable::Table::ColumnFamilyMap.new
```

Comment:
If the constructor is private, should we show it being created?

Reply:
Actually I looked into that, it's not marked private.

Reply:
It was marked `@private`, but this PR currently changes it. Look 4 lines below.

Reply:
Oops, my mistake, thank you for pointing that out. Are you OK with this solution?

Reply:
I am surprised the public api surface is changing to get doctests running. i would prefer to keep the constructor private, not whatevs.

Reply:
Given the follow-up work proposed in #4133 and #4134, I anticipate that users will have no need to create instances of `ColumnFamilyMap`. I propose removing this example now and restoring the `@private` tag on the initializer. Sound OK?

Reply:
Yes, either that, or leave the original example code and skip it until the follow up work is completed.

Reply:
I'll remove the example now. It has very little value since it is just a small fragment of code with no explicit use case (meaning, it does not result in any interaction with the service).

Reply:
Done.